Stereo Approaches (#418)
vatpac-technology/sops: docs/terminal/williamtown.md

† *Non-standard positions* may only be used in accordance with [VATPAC Air Traffic Services Policy](https://vatpac.org/publications/policies){target=new} 

## Airspace
WLM TCU owns the airspace within the WLM MIL CTR `SFC` to `F125`.

> When WLM TCU is active above `F125` by ad-hoc release or NOTAM, **WLM-L_APP** owns the airspace `SFC` to `F125`, while **WLM APP** owns the rest of WLM TCU airspace above `F125`. This may be adjusted as required with coordination.

Reviewer: @alphadelta332 @ST6-Bravo either of you guys know what the upper limit of the WLM TMA is when both positions are online?

Reviewer 2: It's not particularly well documented in the SOPs at the moment, but I don't believe there is a default. It's just whatever additional airspace is released or RAs activated above F125

Reviewer: Yeah ok. I had a look at work today, there's an airspace segment which extends above the eastern TMA block to FL600 but we don't want FL600 TMAs haha. Maybe we could top it out at say FL245?

    It is the responsibility of the WLM TCU controller(s) to negotiate any airspace releases with ARL(All).

### ADC
When WLM ADC is online, they own the airspace within the WLM MIL CTR A (`SFC`-`A050`). This may be amended/released as required between **WLM ADC** and **WLM TCU**.

<figure markdown>
![WLM ADC Airspace](img/wlmadc.png){ width="700" }
  <figcaption>WLM ADC Airspace</figcaption>
</figure>

## Stereo Approaches (Fast Jet Aircraft)
Fast jet aircraft typically transit from M550 to YWLM via Gate 6 at `F125`. When requested, or advised via ATIS, the stereo approach may be utilised to facilitate aircraft recovery.

!!! tip
    See [Controller Skills](../controller-skills/military.md#stereo-approaches) for more details about stereo approaches.

!!! Example
**MAPL45**: "Willy Approach, MAPL45, RTBing via G6, runway 12" 

> **WLM TCU**: "MAPL45, Willy Approach, cleared stereo approach runway 12, via G6, once 9DME north of the TACAN, descend not below 'A050'"  
> **MAPL45**: "Cleared stereo approach runway 12, via G6, once 9DME north of the TACAN, descend not below A050, MAPL45"

Reviewer 3: ```suggestion
**WLM TCU**: "MAPL45, Willy Approach, cleared stereo approach runway 12, via G6, once 9DME north of the TACAN, descend not below `A050`"  
**MAPL45**: "Cleared stereo approach runway 12, via G6, once 9DME north of the TACAN, descend not below `A050`, MAPL45"
```

All other aircraft coming from ARL CTA will be **Heads-up** Coordinated to WLM TCU.

> ### Stereo Approaches (Fast Jet Aircraft)

Reviewer 2: I reckon this is probably more appropriate for a **circuits** or **miscellaneous** section, as opposed to the coordination section

Author: Will move it there

> Fast Jet Aircraft typically arrive back into Willy from Gate 6, at 'F125', as such you can assign them the stereo approach to arrive back into Willy.

Reviewer 2: ```suggestion
Fast Jet Aircraft typically arrive back into YWLM from Gate 6, at `F125`, as such you can assign them the stereo approach to arrive back into YWLM.
```

> !!! Example

Reviewer 2: ```suggestion
!!! phraseology
```

> **MAPL45**: "WLM TCU, RTBing via G6 for RWY12"

Reviewer 2: This one is radio telephony, so the callsign is used rather than the code
```suggestion
**MAPL45**: "Willy Approach, MAPL45, RTBing via G6 for RWY12"
```

> **WLM TCU**: "MAPL45, cleared the stereo appraoch for RWY 12 via G6, once 9DME north of the tacan descend not below `A050`"

Reviewer 2: No need to use the `` formatting for altitudes in phraseology examples
```suggestion
**WLM TCU**: "MAPL45, cleared the stereo approach for RWY 12 via G6, once 9DME north of the TACAN, descend not below A050"
```

> **MAPL45**: "Once 9DME north, descend not below `A050`"

Reviewer 2: ```suggestion
**MAPL45**: "Cleared the stereo approach for RWY 12 via G6, once 9DME north of the TACAN, descend not below A050, MAPL45"
```

> <figure markdown>

Reviewer 2: Could we please remove this diagram, we're just avoiding including any copyright images/documents. Links to them are fine. Cheers

Author: Will do

!!! phraseology
    <span class="hotline">**WLM TCU** -> **WLM ADC**</span>: "via WLM, ZULU, Close formation of 5, do you have any restrictions or requirements?”  
    <span class="hotline">**WLM ADC** -> **WLM TCU**</span>: "ZULU, no restrictions or requirements, A040"  

## Miscellaneous
### Stereo Approaches (Fast Jet Aircraft)

> Fast Jet Aircraft typically arrive back into Willy from Gate 6, at 'F125', as such you can assign them the stereo approach to arrive back into Willy.

Reviewer 2: ```suggestion
Fast Jet Aircraft typically arrive back into YWLM from Gate 6, at `F125`, as such you can assign them the stereo approach to arrive back into YWLM.
```

> !!! Example

Reviewer 2: I think I made some suggestions for this whole section in the first review that haven't come across yet

Author: Yeah, when I do the commit suggestion it appears it doesnt change it will go back and fix it,
